### PR TITLE
Allow closing brackets on the same line

### LINKED ISSUE #13078: Delete your jQuery Functions not passing with correct code
#### Challenge Name
[Delete your jQuery Functions](https://www.freecodecamp.com/challenges/delete-your-jquery-functions#?solution=fccss%0A%24(document).ready(function()%7B%7D)%3B%0Afcces%0A%0A%3C!--%20Only%20change%20code%20above%20this%20line.%20--%3E%0A%0A%3Cdiv%20class%3D%22container-fluid%22%3E%0A%20%20%3Ch3%20class%3D%22text-primary%20text-center%22%3EjQuery%20Playground%3C%2Fh3%3E%0A%20%20%3Cdiv%20class%3D%22row%22%3E%0A%20%20%20%20%3Cdiv%20class%3D%22col-xs-6%22%3E%0A%20%20%20%20%20%20%3Ch4%3E%23left-well%3C%2Fh4%3E%0A%20%20%20%20%20%20%3Cdiv%20class%3D%22well%22%20id%3D%22left-well%22%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target1%22%3E%23target1%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target2%22%3E%23target2%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target3%22%3E%23target3%3C%2Fbutton%3E%0A%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3Cdiv%20class%3D%22col-xs-6%22%3E%0A%20%20%20%20%20%20%3Ch4%3E%23right-well%3C%2Fh4%3E%0A%20%20%20%20%20%20%3Cdiv%20class%3D%22well%22%20id%3D%22right-well%22%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target4%22%3E%23target4%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target5%22%3E%23target5%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target6%22%3E%23target6%3C%2Fbutton%3E%0A%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%3C%2Fdiv%3E%0A%3C%2Fdiv%3E%0A)

#### Issue Description
Correct code does not pass.
It wants }); to be on a new line, which is weird.
First time seeing any1 with this problem so I think this is kind of bike-shredding but I will let you decide

#### Browser Information

* Browser Name, Version: Chrome v56
* Operating System: OS X 10.10
* Mobile, Desktop, or Tablet: 

#### Your Code
```
<script>
$(document).ready(function(){});
</script> 
```
#### Screenshot
image

### PR DESCRIPTION
**Pull request by ajain17**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [x] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13078

### CODE REVIEW

**Review by Greenheart — COMMENTED** [review]
@ajain17 Thanks for the quick fix! :blush:

`.*` works but I'm not sure if `{` might be enough for this test?
Also `\n*` will be covered by `\s*` since newlines are whitespace.

**Comment by dhcodes** [comment]
@Greenheart @ajain17 I was going to go in favor of changing the first test to be: `assert(code.match(/\\{\\s*\\}\\);/g)` to ensure no text is there. Right now the first test checks only that the last few characters of each function are removed.

File: seed/challenges/03-front-end-libraries/jquery.json [hunk 1]
```diff
@@ -421,7 +421,7 @@
         "assert(!code.match(/e\"\\);/g) && !code.match(/t\"\\);/g), 'message: Delete all three of your jQuery functions from your <code>document ready function</code>.');",
```
Comment (dhcodes) [inline seed/challenges/03-front-end-libraries/jquery.json]: While fixing this, change this test to:
`assert(code.match(/\\{\\s*\\}\\);/g)`

**Comment by Greenheart** [comment]
@dhcodes Any updates on this? I think it would be nice against @ajain17 if we could come to a conclusion :wink:

Do we request further changes or can we merge this?

**Comment by ajain17** [comment]
Hi, sorry for the delay. was traveling. will do it today.

**Comment by dhcodes** [comment]
Kind of indifferent. Sort of depends on if you feel the part in the between the `{ }` should be blank

File: seed/challenges/03-front-end-libraries/jquery.json [hunk 2]
```diff
@@ -418,10 +418,10 @@
         "</div>"
       ],
       "tests": [
-        "assert(!code.match(/e\"\\);/g) && !code.match(/t\"\\);/g), 'message: Delete all three of your jQuery functions from your <code>document ready function</code>.');",
+        "assert(code.match(/\\{\\s*\\}\\);/g), 'message: Delete allllll three of your jQuery functions from your <code>document ready function</code>.');",
```
Comment (Greenheart) [inline seed/challenges/03-front-end-libraries/jquery.json]: Delete **allllllll** of them? :wink:

**Comment by raisedadead** [comment]
@Greenheart @dhcodes bump for a review.

**Comment by dhcodes** [comment]
I'm not sure this ever got updated? @ajain17

**Comment by ajain17** [comment]
@dhcodes  I updated it about 16 days back

**Comment by dhcodes** [comment]
Okay. Camperbot was down so it was hard to tell. My bad, will test today. Thanks!

**Comment by dhcodes** [comment]
Tested locally. **LGTM** Thanks for making this fix! 👍  💯  🌮

**Comment by erfe1** [comment]
سوف امسح الايميل

بتاريخ ٢٠١٧/٠٢‏/٢٤ ٧:٢٦ م، كتب "Dylan" <notifications@github.com>:

**Pull request merged**